src/App.tsx

```diff
@@ -16,35 +53,58 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<string>('');
+  const [reverse, setReverse] = useState<boolean>(false);
+
+  const goodsToShow = sortGoods(goodsFromServer, sortField, reverse);
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          onClick={() => setSortField(SORT_BY.name)}
+          className={`button is-info ${sortField === SORT_BY.name ? '' : 'is-light'}`}
```

Comment:
To set classes conditionally use [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx) library